product-service/consts/product-db-queries.ts

```diff
@@ -0,0 +1,3 @@
+export const selectAllProducts = 'select * from products inner join stocks on products.id = stocks.product_id';
+export const selectProductWithSpecifiedId = (id) => `select * from products where id=${id}`
+export const insertProduct = (title, price, description) => `insert into products (id, title, price, description) values (default, '${title}', ${price}, '${description}') returning id`;
```

Comment:
Consider using SQL query params instead of inserting data directly into the query to avoid SQL Injections.

Reply:
Thanks for suggestion about sql injections also thought about it.
Will fix soon all of the comments